src/components/NewSongForm.js

```diff
@@ -11,7 +11,7 @@ class NewSongForm extends Component {
   saveSong (e) {
     e.preventDefault()
     const {title, artist, album, votes} = this.state
-    Firebase.database().ref('songs').push({title, artist, album, votes})
+    this.props.firebase.push('/songs', {title, artist, album, votes})
```

Comment:
would like to hide the implementation details of `saveSong` from this component and make that function available as a prop as was done for the `changeVote` function